packages/common/src/index.js

```diff
@@ -112,8 +112,8 @@ const REPORT_STATUSES = {
 exports.REPORT_STATUSES = REPORT_STATUSES;
 
 const TRAINING_REPORT_STATUSES = {
-  NOT_STARTED: 'not_started',
-  IN_PROGRESS: 'in_progress',
+  NOT_STARTED: 'not-started',
+  IN_PROGRESS: 'in-progress',
```

Comment:
Please use the same values used on other things like AR's or goals. "Not Started" "In Progress"

Reply:
These are for the routes.

Reply:
This change is also not part of this ticket just a artifact from the other PR (should go away once the other is merged).